docs/docs/architecture/security-considerations.md

```diff
@@ -57,10 +57,24 @@ Ideally, a volume is mounted as a raw block device and authenticated encryption
 
 ### Logs
 
-By default, container logs are visible to the host.
-Sensitive information shouldn't be logged.
-
-As of right now, hiding logs isn't natively supported.
-If `ReadStreamRequest` is denied in the policy, the Kata Agent stops reading the logs.
-This causes the pipes used for standard out and standard error to fill up and potentially deadlock the container.
-If absolutely required, standard out and standard error should be manually redirected to `/dev/null` inside the container.
+By default, container logs are visible to the host to enable normal Kubernetes operations, for example debugging using `kubectl logs`.
+The application needs to ensure that sensitive information isn't logged.
+
+If logs access isn't required, it can be denied by setting the `ReadStreamRequest` default to `false` in the `settings.json` file.
```

Comment:
I think it's unclear where in the flow you would do that an how you get this file. We need a separate section on how to do advanced configuration in general, please create a ticket.

Reply:
Added more instructions and created the ticket.